### ISSUE TITLE
Release EXE flagged as malware

### ISSUE THREAD

**Issue by TulioAdriano** [seritools/owowon#5]
It has been flagged by Chrome upon download. When I ignored the warning to download and scanned with Windows Defender, the computer acted very strange, bouncing between chrome windows. I unplugged the PC immediately in case it's been hacked/infected by malware. 

Please rebuild the release, or post build instructions on the Readme.md file. 

I'm looking forward to use this awesome GUI.

**Comment by seritools** [comment]
> It has been flagged by Chrome upon download. 

This is normal for any unsigned binary.

I just downloaded it and ran it again, no weird behavior over here.

The only thing that triggers (bad) AVs is that I've compressed it with UPX: https://upx.github.io/

Feel free to `upx -d owowon-gui.exe` to decompress it and run it through virustotal or anything else. I've also attached it uncompressed here:

[owowon-gui.zip](https://github.com/user-attachments/files/16180659/owowon-gui.zip)
